core/templates/core/includes/articles_list.html

```diff
@@ -14,12 +14,12 @@
         {% else %}
             <div class="col-12">
         {% endif %}
-            <h2>
-                <a href="{% pageurl article %}">{{ article.headline|richtext }}</a>
-            </h2>
-            {% if article.summary %}
-                {{ article.summary|richtext }}
-            {% endif %}
```

Comment:
Also just realized: these indents shouldn't be here. I was messing around with something and forgot to change it back.